package.json

```diff
@@ -34,6 +34,7 @@
     "jsxgettext-andris": "^0.9.0-patch.1"
   },
   "dependencies": {
+    "async": "^2.1.5",
```

Comment:
It's a dependency of jquery-file-upload-middleware: https://github.com/aguidrevitch/jquery-file-upload-middleware/blob/master/package.json#L17 but `npm install` didn't work for me.